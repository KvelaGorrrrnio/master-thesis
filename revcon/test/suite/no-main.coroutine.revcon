@coroutine

coroutine this_is_not_main() {
  a += 1
}
