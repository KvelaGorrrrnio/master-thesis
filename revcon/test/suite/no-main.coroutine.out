No main coroutine was found.
